File: task-03/Go/Subtask2.go
package main // This tells the program it's the main package

import (
    "io/ioutil" // This lets us work with files
)

func main() {
    // Read the content of "input.txt" into a variable called "data"
    data, _ := ioutil.ReadFile("input.txt")

    // Write the data into the file "output.txt"
    _ = ioutil.WriteFile("output.txt", data, 0644)
}

File: task-03/Rust/Subtask2.rs
use std::fs; // This lets us work with files

fn main() {
    // Read the content of "input.txt" into a variable called "data"
    let data = fs::read_to_string("input.txt").expect("Unable to read file");

    // Write the data into the file "output.txt"
    fs::write("output.txt", data).expect("Unable to write file");
}

